vignettes/ees-api-workflow.Rmd

```diff
@@ -174,23 +174,28 @@ eesyapi::get_meta(dataset_id = "7c0e9201-c7c0-ff73-bee4-304e731ec0e6") |>
 
 ## Retrieving data from a data set
 
-### Downloading an entire data set in a single file
+### Preview raw CSV data
```

Comment:
Makes sense here thematically, but just wondering if actually in terms of a real user's journey, whether the data preview bit would be better being put before the meta data query? Feel like that was part of what they were getting at in the session yesterday.

Reply:
Moved upwards in PR response commit